src/main/java/net/minecraftforge/event/entity/living/potions/PotionEvent.java

```diff
@@ -0,0 +1,22 @@
+package net.minecraftforge.event.entity.living.potions;
+
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.potion.PotionEffect;
+import net.minecraftforge.event.entity.living.LivingEvent;
+import net.minecraftforge.fml.common.eventhandler.Cancelable;
+
+/** Event for when a potion */
+@Cancelable
+public class PotionEvent extends LivingEvent {
```

Comment:
I would suggest calling this `PotionEffectEvent`. `PotionEffect` and `Potion` are different classes, and if events for `Potion` were added, there would be a name conflict.